src/views/privacy-policy.njk

```diff
@@ -9,10 +9,12 @@
 
       <h1 class="govuk-heading-l">Privacy Policy for Gulls (Public Health and Safety)</h1>
 
-      <p class="govuk-body">This policy applies to the content of the Licence Application available at:</p>
+      <p class="govuk-body">This policy applies to the content of the online applications available at:</p>
 
       <p class="govuk-body"><a class="govuk-link" href="https://licensing.nature.scot/gulls-health-and-safety/">https://licensing.nature.scot/gulls-health-and-safety/</a></p>
 
+      <p class="govuk-body"><a class="govuk-link" href="https://licensing.nature.scot/gulls-health-and-safety-return/">https://licensing.nature.scot/gulls-health-and-safety/</a></p>
```

Comment:
the link text should say

https://licensing.nature.scot/gulls-health-and-safety-return/